### PR TITLE
feat: #3238 enhance ui

### PR DESCRIPTION
**Pull request by Bdragon93**
# Pull request checklist

feat: #3238

**Any additional notes for reviewer (optional):**

### CODE REVIEW

File: packages/reapit-connect-management/public/index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <base href="/" />
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" />
-    <title>Foundations App</title>
+    <title>Reapit Management</title>
```
Comment (willmcvay) [inline packages/reapit-connect-management/public/index.html:7]: Should be Reapit Connect Management

File: packages/reapit-connect-management/src/components/ui/offices/edit-office-group.tsx [hunk 2]
```diff
@@ -80,9 +80,14 @@ export const UpdateOfficeGroupModal: React.FC<UpdateOfficeGroupModalProps> = ({
...
-      <H5>{`Editing ${editingGroup.name}`}</H5>
-      <p>
+    <ModalV2
+      visible={!!editingGroup}
+      destroyOnClose={true}
+      onClose={handleOnClose}
+      title={`Editing ${editingGroup.name}`}
+      zIndex={90}
+    >
+      <p className="hepler-text">
```
Comment (willmcvay) [inline packages/reapit-connect-management/src/components/ui/offices/edit-office-group.tsx:90]: Should be helper-text - typo!

File: packages/reapit-connect-management/src/components/ui/offices/offices-groups-tab.tsx [hunk 3]
```diff
@@ -84,10 +88,10 @@ const OfficesGroupsTab: React.FC = () => {
       <Section>
         <div className={tabTopContent}>
           <div className={tableTitle}>
-            <H5>Existing office groups</H5>
+            <H5>Office groups</H5>
             <Button onClick={onOpenCreateModel}>Create office group</Button>
           </div>
-          <p>
+          <p className="hepler-text">
```
Comment (willmcvay) [inline packages/reapit-connect-management/src/components/ui/offices/offices-groups-tab.tsx:94]: As above, typo

File: packages/reapit-connect-management/src/components/ui/offices/offices-tab.tsx [hunk 4]
```diff
@@ -67,29 +65,28 @@ const OfficesTab: React.FC = () => {
...
+    { Header: 'Last Updated', Cell: LastUpdatedCell },
   ]
 
   return (
     <ErrorBoundary>
       <Section>
         <div className={tabTopContent}>
           <H5 className={tableTitle}>Existing offices</H5>
-          <p>
+          <p className="hepler-text">
```
Comment (willmcvay) [inline packages/reapit-connect-management/src/components/ui/offices/offices-tab.tsx:85]: Same typo!

File: packages/reapit-connect-management/src/components/ui/users/edit-user-group.tsx [hunk 5]
```diff
@@ -0,0 +1,135 @@
+import React, { useCallback } from 'react'
```
Comment (willmcvay) [inline packages/reapit-connect-management/src/components/ui/users/edit-user-group.tsx:1]: Looks like this file is missing a test file

File: packages/reapit-connect-management/src/components/ui/users/user-groups-tab.tsx [hunk 6]
```diff
@@ -35,20 +35,31 @@ const UserGroupsTab: React.FC = () => {
...
   ]
 
   return (
     <ErrorBoundary>
       <Section>
         <div className={tabTopContent}>
-          <H5 className={tableTitle}>Existing user groups</H5>
-          <p>
+          <H5 className={tableTitle}>User groups</H5>
+          <p className="hepler-text">
```
Comment (willmcvay) [inline packages/reapit-connect-management/src/components/ui/users/user-groups-tab.tsx:62]: Typo!

File: packages/reapit-connect-management/src/components/ui/users/users-tab.tsx [hunk 7]
```diff
@@ -71,7 +71,7 @@ const UsersTab: React.FC = () => {
       <Section>
         <div className={tabTopContent}>
           <H5 className={tableTitle}>Existing users</H5>
-          <p>
+          <p className="hepler-text">
```
Comment (willmcvay) [inline packages/reapit-connect-management/src/components/ui/users/users-tab.tsx:74]: And again!

**Review by willmcvay — CHANGES_REQUESTED** [review]
Just a few small things, others LGTM

File: packages/reapit-connect-management/public/index.html (hunk 1 above)
Comment (Bdragon93) [reply packages/reapit-connect-management/public/index.html:7]: - `Rename tab name from 'Foundations App' to ‘Reapit Management’` in ticket's description cc @HollyJoyPhillips

**Review by willmcvay — APPROVED** [review]
Thanks 👍

File: packages/reapit-connect-management/src/components/ui/users/edit-user-group.tsx (hunk 5 above)
Comment (Bdragon93) [reply packages/reapit-connect-management/src/components/ui/users/edit-user-group.tsx]: did not add before commit but `pre-commit` did it :D

**Pull request merged**